core/src/parser/utils.rs

```diff
@@ -845,18 +845,8 @@ pub fn strip_indent(mut chunks: Vec<StrChunk<RichTerm>>) -> Vec<StrChunk<RichTer
     chunks
 }
 
-/// Strip the indentation of a doc metadata. Wrap it as a literal string chunk and call
-/// [`strip_indent`].
-pub fn strip_indent_doc(doc: String) -> String {
```

Comment:
I guess this was just an unused function, that wasn't flagged such because of how LALRPOP works?

Reply:
It wasn't flagged because it's declared as `pub`. But it's not used anywhere in the repository.

Reply:
Hm, I wonder if this is the case of many more such functions, as we flag quite a bunch of things with `pub`. There's probably a flag somewhere to say "flag unused stuff even if it's pub"...